webui/react/src/components/Table.tsx

```diff
@@ -104,7 +104,7 @@ export const taskIdRenderer: TaskRenderer = (_, record) => (
       <ConditionalWrapper
         condition={canBeOpened(record)}
         wrapper={children => (
-          <Link path={waitPageUrl(record)}>
+          <Link path={paths.interactiveTask(record, waitPageUrl(record))}>
```

Comment:
**suggestion:** since the 2nd param `waitPageUrl(record)` is a derivative of the 1st param `record`, how about just taking in the 1st param and then calculate the task url from it within `paths.interactiveTask()`?